sensor_msgs/Image sonar_scan_1
sensor_msgs/Image sonar_scan_2
float64 size_of_pixel
float64[16] initial_guess
---
fms2d/PotentialSolution potential_solution